index.bs

```diff
@@ -2858,17 +2858,16 @@ the inputs to the algorithm needed to compute the set.
 An effective overload set is used, among other things, to determine whether there are ambiguities in the
 overloaded operations, constructors and callers specified on an interface.
 
-The elements of an effective overload set are tuples of the form
-&lt;|callable|, |type list|, |optionality list|&gt;.  If the effective overload
+The elements of an effective overload set are [=tuples=] of the form
+(|callable|, |type list|, |optionality list|).  If the effective overload
```

Comment:
Also, a lot of places seem to refer to "type list length" and similar; we should ideally switch it to size, and link size.